geometry_msgs/Pose[] poses
duration time_from_start